com.atlauncher.ATLauncher.metainfo.xml

```diff
@@ -0,0 +1,122 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html -->
+<!-- Credits to github/nickavem for the app data, Doomsdayrs for edits -->
+<component type="desktop">
+    <id>com.atlauncher.ATLauncher</id>
+    <metadata_license>MIT</metadata_license>
+    <name>ATLauncher</name>
+    <summary>A launcher for Minecraft which integrates multiple different modpacks to allow you to download and install modpacks easily and quickly.</summary>
+    <icon type="remote" width="500" height="500">https://atlauncher.com/assets/images/logo.png</icon>
+    <description>
+        <p>ATLauncher is a launcher for Minecraft which integrates multiple different modpacks to allow you to download and install modpacks easily and quickly.</p>
+        <p>ATLauncher has a few nice features including:</p>
+        <ul>
+            <li>Allows quick and easy install of modpacks, no more messing around trying to install mods</li>
+            <li>A range of different packs, and more being added all the time</li>
+            <li>Multiple instances. Install a pack as many times as you like without messing up a previous install</li>
+            <li>Multiple accounts allow you to login as different users from the one install</li>
+            <li>Private instances. This allows you to only install instances under your account so people can't mess up your worlds</li>
+            <li>1 click backup of your saves</li>
+            <li>Easy server creation from within the Launcher</li>
+            <li>Install packs from CurseForge and FTB</li>
+            <li>Mod management for instances, including auto download and installation of mods, maps and resource packs from CurseForge and Modrinth</li>
+            <li>Support for both Mojang and Microsoft accounts</li>
+            <li>More features always being added</li>
+        </ul>
+    </description>
+    <categories>
```

Comment:
There's no need to define it here, it will be imported from the desktop file.

Reply:
What?

Reply:
I do not understand what you mean by that

Reply:
Categories are imported from the desktop file. There's no need to define them explicitly in the appdata file.

Reply:
Oh! Where is that documented? I did not know of that till now @_@

Reply:
That should do it.